packages/core/admin/server/services/user.js

```diff
@@ -226,7 +237,13 @@ const deleteById = async (id) => {
     }
   }
 
-  return strapi.query('admin::user').delete({ where: { id }, populate: ['roles'] });
+  const deletedUser = await strapi
+    .query('admin::user')
+    .delete({ where: { id }, populate: ['roles'] });
+
+  strapi.eventHub.emit('user.delete', sanitizeUser(deletedUser));
```

Comment:
ℹ️ Actually I don't think `deleteById` is triggered via the admin panel, only `deleteByIds` so maybe I should remove the emit here?

Reply:
Since the service to delete a single one already exists I think it's good to track it just in case